### ISSUE TITLE
Can't find Elastix functions after a sucessful install

### ISSUE THREAD

**Issue by jeekiii** [SuperElastix/SimpleElastix#159]
I sucessfully installed simpleElastix using the method described in the doc, I ran the needed "sudo python setup.py install" seemingly successfully, Yet when I import SimpleITK (which work) there is no Elastix anywhere. It returns "AttributeError: module 'SimpleITK' has no attribute 'Elastix'". Looking at the available functions and classes from SimpleITK there is nothing from Elastix at all.

At this point I'm pretty lost. 

I have python3.6.3 and arch linux.

**Comment by kaspermarstal** [comment]
The most likely explanation is that you installed SimpleElastix in another Python environment than the one from which you are running `import SimpleITK` _and_ that there is an existing installation of SimpleITK in that environment. Try to make a virtual environment and run `sudo python setup.py install` here, and then try to import it.

**Comment by jeekiii** [comment]
Thanks it actually works, can't believe I didn't think of that, I feel pretty dumb for not trying it given how simple it was.

What's weird to me is that I can't find any version of python with Elastix working (outside of the virtualenv I just created). Is it possible that it installed it into a virtualenv by default or something?

**Comment by kaspermarstal** [comment]
What the python pointed to by `which python`? 

The naming is confusing and the package will be renamed from SimpleITK to SimpleElastix very soon.

**Comment by jeekiii** [comment]
The confusing part to me was that it's the same python version in the virtual environment (3.6) and out of it, yet it only works if I install simpleElastix inside the virtual environment.

Sorry for not responding for so long, I had forgotten about this issue.

I realize my previous message isn't very clear: I tried a bunch of python versions (3.5, 3.6, and some variant of python2) and none of it worked, then I tried as you suggested inside a virtual environment and it worked.

`< which python >` outside of the virtual environment returns /usr/bin/python, which is version 3.6.4
in the virtual env it's also 3.6.4 (but which python obviously returns /home/.../virtualenv/bien/python). I'm pretty sure it always defaults to /usr/bin/python unless I'm in a virtual environment, I don't see how it could have been installed in another python environment.
